extensions/wikia/Email/tests/EmailIntegrationTest.php

```diff
@@ -16,15 +19,24 @@ function setUp() {
 	 * Make sure all the images we're using exist
 	 */
 	public function testEmailImages() {
-		$this->markTestSkipped();
 
-		$icons = Email\ImageHelper::getIconInfo();
+		$this->setVignetteEnvToProd();
 
-		foreach ( $icons as $info ) {
-			$url = $info['url'];
-			$name = $info['name'];
+		foreach ( Email\ImageHelper::getIconInfo() as $iconInfo ) {
+			$url = $iconInfo['url'];
+			$name = $iconInfo['name'];
 			$response = HTTP::get( $url );
 			$this->assertTrue( $response !== false, "{$name} should return HTTP 200" );
 		}
 	}
+
+	/**
+	 * Set the Vignette base URL to point to production. The testEmailImages test above
+	 * was failing frequently in dev and since prod images are what our user's see as
+	 * part of our emails, we're updating that test to only run in prod. See SOC-860.
+	 */
+	private function setVignetteEnvToProd() {
+		global $wgVignetteUrl;
+		$wgVignetteUrl = self::VIGNETTE_BASE_URL_PROD;
```

Comment:
Please mock the global variable instead of polluting the global scope for next tests.